package.json

```diff
@@ -15,6 +15,7 @@
   },
   "license": "",
   "dependencies": {
+    "@citeproc-rs/wasm": "^0.0.0-canary-0157c4b",
```

Comment:
I have just published [v0.1.0](https://github.com/zotero/citeproc-rs/releases/tag/wasm-v0.1.0) to [npm](https://www.npmjs.com/package/@citeproc-rs/wasm) as we speak. It doesn't include anything new, so it's just everything in master at the moment. New PRs merged will get new version bumps so you'll be able to tell the versions apart.